boards/arm/v2m_musca/dts.fixup

```diff
@@ -0,0 +1,82 @@
+/* SoC level DTS fixup file */
```

Comment:
I am wondering, @galak : wouldn't it be more readable, if we had 2 different fixup files, for Secure and for Non-Secure hardware description?

Reply:
Actually: would it be possible?